app/src/main/java/com/amaze/filemanager/ui/fragments/FtpServerFragment.kt

```diff
@@ -584,6 +587,33 @@ class FtpServerFragment : Fragment(R.layout.fragment_ftp) {
             .append(pathToDisplayString(defaultPathFromPreferences))
         if (readonlyPreference) sb.append(" \uD83D\uDD12")
         sharedPath.text = sb.toString()
+        setListener()
+    }
+
+    private fun setListener() {
+        sharedPath.setOnTouchListener { _, event ->
```

Comment:
Any specific reason for it to be on touch listener and not on click listener? Please fix this later.

Reply:
> Any specific reason for it to be on touch listener and not on click listener? Please fix this later.

@VishalNehra Sorry, seeing this now. The reset button is a `endIconDrawable` to the `TextView` (`sharedPath`). And this is the only way to set a click listener callback to it.